dbt_platform_helper/templates/pipelines/codebase/overrides/stack.ts

```diff
@@ -432,10 +432,14 @@ export class TransformedStack extends cdk.Stack {
     }
 
     private getEnvManagerRolePolicyDoc() {
+        const accounts = this.pipelinesFile.accounts ?? [this.account]
+
         return {
             Effect: 'Allow',
             Action: ['sts:AssumeRole'],
-            Resource: [`arn:aws:iam::${this.account}:role/${this.appName}-*-EnvManagerRole`],
+            Resource: accounts.map(
+                account => `arn:aws:iam::${account}:role/${this.appName}-*-EnvManagerRole`
+            ),
```

Comment:
I know this file is not tested, and is hopefully going away soon, but how have you tested this change? What manual steps have you taken to ensure it works?

Reply:
@jackvdp Question above still applies. Would you be able to demo it working to some of us in the platform team?